src/components/editor/plugins/assembly/assembly.grammar

```diff
@@ -87,6 +88,18 @@ BranchInstructionWithoutLabel {
   RegisterLiteral
 }
 
+DataInstructionDeclaration {
+    Label (kw<"DCD"> | kw<"DCW"> | kw<"DCB">) IntegerLiteral
```

Comment:
Add possiblity for String Literal

Reply:
as discussed closed

Reply:
As discussed with Leo and Michael, we will not implement this.